src/res/translation/wininstaller/fr.nsi

```diff
@@ -0,0 +1,37 @@
+; French translation
+
+LangString DESKTOP_SET_SHORTCUT ${LANG_FRENCH} \
+    "Créer un raccourci sur le bureau"
+
+LangString INVALID_FOLDER_MSG ${LANG_FRENCH} \
+    "Le dossier de destination existe déjà. Veuillez entrer un nouveau dossier de destination."
+
+LangString RUNNING_APP_MSG ${LANG_FRENCH} \
+    "${APP_NAME} est en cours d'exécution. Veuillez le fermer et relancer l'installation."
+
+LangString OLD_WRONG_VER_FOUND ${LANG_FRENCH} \
+    "Nous avons détecté une ancienne version de $ {APP_NAME} dans votre dossier Program Files 32 bits. Il est fortement recommandé de le supprimer avant d'installer une nouvelle version de $ {APP_NAME}. Voulez-vous le supprimer maintenant ? "
```

Comment:
I think you need to remove the space between $ and {APP_NAME}

Reply:
Thanks for reviewing. I do those changes.